### DISCUSSION TITLE
Registering Images in Plugin Pipeline Simulation

### DISCUSSION THREAD

**Discussion by bwmodular** [jordimontana82/fake-xrm-easy#575] [General]
Am I right in thinking that this isn't currently supported, pending merging of this PR?
https://github.com/jordimontana82/fake-xrm-easy/pull/496

Not a problem - just wanting to check I'm not missing anything....

**Comment by jordimontana82** [comment]
Hi Ben, correct, hasn't been merged yet.... I'll reference it in the other issue you raised yesterday
